fs-one-concepts.md

```diff
@@ -3,19 +3,40 @@
 Try your best to answer each question on your own before looking up the answer online. Once you're done writing your first answer, you can google the question and write the best answer you find.
 
 #### 1. What is Enzyme and what are some of the methods that it provides?
+Enzyme is a JavaScript Testing utility for React that makes it easier to test your React Components' output. 
+You can also manipulate, traverse, and in some ways simulate runtime given the output. 
 
 #### 2. What is the difference between dynamic and a static routes?
+Static routes allows for inspection and matching of routes before rendering. It is very useful on the server side.
+Takes place as your app is rendering, not in a configuration or convention outside of a running app. ":id" part of the paths in those Route compnonents.
 
 #### 3. What is a JSON API?
+JSON API is a format that works with HTTP
 
 #### 4. What is a migration and why would you use one?
-
+Migrations are files that run SQL commands for us.
+Maintain consistency between the database you run locally for development.
 #### 5. Explain how to set up a route in React.
+npm install react-router-dom
+import ReactDOM from 'react-dom'
+import { Route, Link, BrowserRouter as Router } from 'react-router-dom'
+Render router
 
 #### 6. When would you use a generate resource over a generate controller?
+generate resource gives us a model file and a migration file. 
```

Comment:
I'd like to stretch you here a bit. What does a generate resource add to the routes.rb?